base_multi_image/models/image.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+# © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+#        Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+import urllib
```

Comment:
```suggestion
from urllib.request import urlretrieve
```
`urllib.urlretrieve(url)` won't work in python 3